CMakeLists.txt

```diff
@@ -77,27 +79,39 @@ if( BUILD_APPLICATIONS )
   add_subdirectory( app )
 endif()
 
-export( TARGETS IntersonArrayCxx
-  FILE "${PROJECT_BINARY_DIR}/IntersonArraySDKCxxTargets.cmake" )
-export( PACKAGE IntersonArraySDKCxx )
+# Configure 'IntersonArraySDKCxxConfig.cmake' for a build tree
+set(INCLUDE_DIR_CONFIG ${INCLUDE_BUILD_DIR})
+set(intersonArraySDKCxx_config ${PROJECT_BINARY_DIR}/IntersonArraySDKCxxConfig.cmake)
+configure_package_config_file(
+    IntersonArraySDKCxxConfig.cmake.in
+    ${intersonArraySDKCxx_config}
+    INSTALL_DESTINATION ${PROJECT_BINARY_DIR}
+    PATH_VARS CONF_INCLUDE_DIRS
```

Comment:
The variable is set to two different values for a build tree an install tree. So, the setting removed on line 90 needs to be added before `# Configure 'IntersonArraySDKCxxConfig.cmake' for an install tree`